### ISSUE TITLE
HotReload other types [Epic]

### ISSUE THREAD

**Issue by MiguelMadero** [toranb/ember-cli-hot-loader#6]
This is an epic and needs to be broken down as we start progress, it's here just as a reminder and reference for others. The details about the different types is on [usecanvas](https://usecanvas.com/mmadero/hotreload-brain-dump-types/0bIUkoYx0uooJHGcrGoyPP).

**Comment by MiguelMadero** [comment]
Can we reopen this? I think I linked to the wrong issue from #10

**Comment by toranb** [comment]
done ;)

**Comment by toranb** [comment]
Support for additional types finally landed to day in v0.4.0

You configure the specific folder names like so from your app config

```javascript
//my-app/config/environment.js
ENV['ember-cli-hot-loader'] = {
  supportedTypes: ['components', 'reducers']
}
```
